src/main/java/datawave/data/type/ListType.java

```diff
@@ -0,0 +1,40 @@
+package datawave.data.type;
+
+import datawave.data.normalizer.Normalizer;
+import datawave.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListType extends BaseType implements OneToManyNormalizerType {
```

Comment:
Should this be abstract and then create a LcNoDiacriticsListType ?

Reply:
I do like that idea better....